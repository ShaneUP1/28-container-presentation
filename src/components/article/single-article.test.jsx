import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Article from './single-article';

describe('single-article component', () => {
  afterEach(() => cleanup());
  it('renders a single article', () => {
    const { asFragment } = render(<Article
      title="Your Title Here"
      author="Jack Handy"
      description="some deep thoughts by Jack Handy"
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
