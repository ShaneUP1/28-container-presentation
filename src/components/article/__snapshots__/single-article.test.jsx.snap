// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`single-article component renders a single article 1`] = `
<DocumentFragment>
  <div>
    <h2>
      Your Title Here
    </h2>
    <h3>
      Jack Handy
    </h3>
    <p>
      some deep thoughts by Jack Handy
    </p>
  </div>
</DocumentFragment>
`;
